pkg/systemd/services.html

```diff
@@ -62,6 +62,19 @@
       <div class="filter-group">
         <button id="create-timer" data-toggle="#timer-dialog" data-target="#timer-dialog" class="btn btn-primary" translate>Create Timer</button>
         <input type="search" id="services-text-filter" class="form-control" aria-label="Filter" placeholder="Filter by Id or description..." translate="placeholder"/>
+        <div class="dropdown" id="services-dropdown">
```

Comment:
FTR, in the spirit of #11155 this should become an actual `<select>` soon. Right now the selector isn't keyboard navigatable at all. Fine for a follow-up of course.